packages/apm/src/hubextensions.ts

```diff
@@ -27,6 +27,18 @@ function traceHeaders(this: Hub): { [key: string]: string } {
  * @param context Properties with which the span should be created
  */
 function startSpan(this: Hub, context: SpanContext | TransactionContext): Transaction | Span {
+  /**
+   * @deprecated
+   * We have this here as a fallback to not break users upgrading to the new API
```

Comment:
Does this work in the middle of the func body? Did you mean to put if above the func signature?

Reply:
It's more a marker for us once we have a major bump to delete this block and document the breaking change.